lib/dispatcher/client.js

```diff
@@ -296,8 +296,7 @@ class Client extends DispatcherBase {
   }
 
   [kDispatch] (opts, handler) {
-    const origin = opts.origin || this[kUrl].origin
```

Comment:
I'm not sure why did we have this set here if clients maps to single origin (single socket); it was causing to request have different origin than the actual client creating the request.

Tests does not seems to show a problem removing it.

Reply:
what does git blame say?

Reply:
Haven checked, let me verify